code/datums/elements/decal.dm

```diff
@@ -76,3 +90,11 @@
 	SIGNAL_HANDLER
 
 	examine_list += description
+
+/datum/element/decal/proc/shuttle_move_react(datum/source, turf/newT)
+	SIGNAL_HANDLER
+
+	if(newT == source)
+		return
+	Detach(source)
+	newT.AddElement(/datum/element/decal, _dir = pic.dir, _cleanable = cleanable, _description = description, _pic = pic)
```

Comment:
If you feel bad about transferring old pic instead of creating clear new with old vars, i can revert this to work like in `rotate_react`.